library/ssl_tls.c

```diff
@@ -583,10 +583,10 @@ static void ssl_update_checksum_start( mbedtls_ssl_context *, const unsigned cha
 static void ssl_update_checksum_md5sha1( mbedtls_ssl_context *, const unsigned char *, size_t );
 #endif
 
-#if defined(MBEDTLS_SSL_PROTO_SSL3)
+#if defined(MBEDTLS_SSL_PROTO_SSL3) && !defined(MBEDTLS_DEPRECATED_REMOVED)
 static void ssl_calc_verify_ssl( mbedtls_ssl_context *, unsigned char * );
 static void ssl_calc_finished_ssl( mbedtls_ssl_context *, unsigned char *, int );
```

Comment:
I did not add the MBEDTLS_DEPRECATED_WARNING here because the user would just get a couple of "random" warnings that mention internal functions instead of saying "SSLv3 is deprecated"